app/views/groupTab/components/MemberList.js

```diff
@@ -1,25 +1,44 @@
-'use strict'
-import React from 'react'
-import { Text, Button, Container, Content, List, ListItem } from 'native-base'
-import { connect } from 'react-redux'
+'use strict';
+import React from 'react';
+import { Text, Button, Container, Content, List, ListItem } from 'native-base';
+import { connect } from 'react-redux';
+import { fetchGroupUsers } from '../../../redux/reducers/users';
+
+class UserList extends React.Component {
+  componentDidMount() {
+    this.props.fetchGroupUsers();
+  }
 
-export default class TaskList extends React.Component {
   render() {
+    const { allUsers } = this.props;
     return (
       <Container>
         <Content>
           <List>
-            <ListItem>
-              <Text>MEMBER NAME</Text>
-              <Text>IMAGE</Text>
-            </ListItem>
-            <ListItem>
-              <Text>MEMBER NAME</Text>
-              <Text>IMAGE</Text>
-            </ListItem>
+            {allUsers.length ?
```

Comment:
the ternary is not necessary in a connected component when we have the correct type of initialState.  Please remove the ternary and make sure the mapState function handles subscribing to the store appropriately